cluster/local/minikube.sh

```diff
@@ -67,7 +67,6 @@ case "${1:-}" in
     ${HELM} init --service-account tiller
     kubectl -n kube-system rollout status deploy/tiller-deploy
     kubectl -n kube-system rollout status deploy/nginx-ingress-controller
-    kubectl -n kube-system rollout status deploy/default-http-backend
```

Comment:
what is the reason for deleting this?

Reply:
I believe the `default-http-backend` is not needed due to the installation of `nignx-ingress-controller`.